tests/ut/backends/test_redis.py

```diff
@@ -377,3 +377,15 @@ def test_build_key_double_dot(self, set_test_namespace, redis_cache, namespace,
 
     def test_build_key_no_namespace(self, redis_cache):
         assert redis_cache.build_key(pytest.KEY, namespace=None) == pytest.KEY
+
+    @pytest.mark.asyncio
+    async def test_clear_cache_self_namespace(self, redis_pool):
```

Comment:
why do we need two pools? Also this doesn't exactly check that the clear behaves as expected

Reply:
Ooooh true, I misunderstood the whole thing. Will re do

Reply:
Pushing this change now